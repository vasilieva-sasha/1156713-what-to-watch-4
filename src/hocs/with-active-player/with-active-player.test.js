import React from "react";
import renderer from "react-test-renderer";
import withActivePlayer from './with-active-player';

const mock = {
  title: ``,
  poster: ``,
  preview: ``
};

const MockComponent = () => <div />;

const MockComponentWrapped = withActivePlayer(MockComponent);

describe(`WithActivePalyer`, () => {
  it(`WithActivePalyer renders correctly`, () => {
    const tree = renderer.create((
      <MockComponentWrapped film={mock} onCardClick={() => {}}/>
    ), {
      createNodeMock() {
        return {};
      }
    }).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
