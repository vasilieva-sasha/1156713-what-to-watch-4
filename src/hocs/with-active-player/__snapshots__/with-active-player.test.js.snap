// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithActivePalyer WithActivePalyer renders correctly 1`] = `<div />`;
